src/main/scala/essential_scala/Ex3Classes.scala

```diff
@@ -17,9 +17,117 @@ object Ex3Classes extends App {
     def willServe(cat: Cat1) = cat.food == "Chips"
   }
 
+  val eastwood = new Director("Clint", "Eastwood", 1930)
```

Comment:
Why types are declared after exercises? And some exercise start mark comments are missing

Reply:
I guess it's because of the same types used in different exercises